src/ansys/fluent/core/rpvars.py

```diff
@@ -4,10 +4,12 @@
 variables like rpvars but instead through the high-level object-based
 interfaces: solver settings objects and task-based meshing workflow.
 """
-
-from typing import Any
+from typing import Any, List
 
 import ansys.fluent.core.filereader.lispy as lispy
+from ansys.fluent.core.services.field_data import allowed_name_error_message
```

Comment:
We ought to move allowed_name_error_message out of field_data to a common place

Reply:
yes, shall we create a new file and move it or is there a common place for all these?